lib/index.js

```diff
@@ -33,7 +33,8 @@ module.exports = function rewireify(file, options) {
 
   function end() {
     post += "/* This code was injected by Rewireify */\n";
-    post += "if (Object.isExtensible(module.exports)) {\n";
+    post += "if ('functionobject'.match(typeof module.exports) && \n"
```

Comment:
`'functionobject'` is not a JS primitive that I'm aware exists. Something like this may be safer:

```
Object.prototype.toString.call(module.exports) === '[object Object]'
```